components/bookshelf/LogoutBox.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useContext } from "react"
+import { UserContext } from "../../context/UserContext"
+import { LogoutContainer } from "../styled/bookshelf"
+import { useRouter } from "next/router"
+import { Button } from "../styled"
+import { logout } from "../../api/logout"
+
+const LogoutBox = () => {
+  const context = useContext(UserContext)
+  const router = useRouter()
+  if (!context) {
+    return null
+  }
```

Comment:
When would `contexst` be null?

Reply:
its because of this error 
![image](https://user-images.githubusercontent.com/91857581/188649846-332b7ce0-c1ab-4f39-adc1-41693b3acc1c.png)
![image](https://user-images.githubusercontent.com/91857581/188649867-ef951667-e976-47ca-94e4-8af44044206e.png)
on Context
I looked it up and a solution I found is this right [here](https://stackoverflow.com/questions/64503671/getting-ts-error-property-user-does-not-exist-on-type-iauth-null)
![image](https://user-images.githubusercontent.com/91857581/188650214-44203260-f9bd-437a-a063-89b06b6e37e5.png)